asynch/pool.py

```diff
@@ -261,30 +262,36 @@ async def _create_connection(self) -> None:
         if pool_size == maxsize:
             raise AsynchPoolError(f"{self} is already full")
         if pool_size > maxsize:
-            raise RuntimeError(f"{self} is overburden")
+            raise AsynchPoolError(f"{self} is overburden")
 
         conn = await connect(**self._connection_kwargs)
         self._free_connections.append(conn)
 
     async def _acquire_connection(self) -> Connection:
         if not self._free_connections:
-            raise AsynchPoolError(f"no free connection in the {self}")
+            raise AsynchPoolError(f"no free connection in {self}")
 
         conn = self._free_connections.popleft()
         self._acquired_connections.append(conn)
         return conn
 
     async def _release_connection(self, conn: Connection) -> None:
         if conn not in self._acquired_connections:
-            raise AsynchPoolError(f"the connection {conn} does not belong to the {self}")
+            raise AsynchPoolError(f"the connection {conn} does not belong to {self}")
 
         self._acquired_connections.remove(conn)
         self._free_connections.append(conn)
 
-    async def _fill_with_connections(self, n: Optional[int] = None) -> None:
-        to_create = n if n else self.minsize
+    async def _init_connections(self, n: Optional[int] = None) -> None:
+        to_create = n if n is not None else self.minsize
         if to_create < 0:
-            raise ValueError(f"cannot create negative connections ({to_create}) for {self}")
+            msg = f"cannot create ({to_create}) negative connections for {self}"
+            raise ValueError()
+        if to_create == 0:
+            return
+        if (self.connections + to_create) > self._maxsize:
```

Comment:
nit: Here & below: rest of the code except `__repr__` seems to utilise `.minsize` & `.maxsize` properties in favor of the private `._minsize` & `._maxsize` respectfully